zerver/models/realm_audit_logs.py

```diff
@@ -119,6 +119,8 @@ class AuditLogEventType(IntEnum):
 
     SAVED_SNIPPET_CREATED = 800
 
+    CUSTOM_EMAIL_SENT = 810
```

Comment:
This seems like a fine choice, but is definitely worth communicating on, since it diverges from the issue -- it's slightly annoying to change these values, and we like to leave plenty of space between sections.

(The issue said 800, and that's taken)

Reply:
I think 810 is good -- just a comment on discussing things like this proactively in the future.